githook/prepare-commit-msg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import sys
+
+def is_integer(value: str) -> bool:
+    try:
+        int(value)
+        return True
+    except ValueError as _:
+        pass
+
+    return False
+
+def check_commit_message() -> int:
```

Comment:
Bad function name. What exactly does it check?

Reply:
It's Python. I used the default pep8 style guide

Reply:
how would you name it?

Reply:
I would name it check commit message 😆

Reply:
`is_message_contain_issue_id` or something similar